docs/detections/rules-ui-create.asciidoc

```diff
@@ -276,6 +349,27 @@ image::images/rule-actions.png[]
 * Continue with <<rule-notifications>>.
 * Create the rule (with or without activation).
 
+[discrete]
+[[preview-rules]]
+=== Preview your rule (optional)
+
+You can preview a custom query, threshold, or EQL (Event Correlation) rule to get feedback on how noisy the rule will be before submitting it. This allows you to fine-tune the query, if needed, to reduce the amount of alerts that may come in.
+
+To preview a rule:
+
+. Write the rule query.
++
+. Select a timeframe of data to preview query results -- `Last hour` or `Last day` -- from the *Quick query preview* drop-down.
```

Comment:
Note that the `Last month` option is also available in the Preview section for `Custom Query` and `Threshold` rules.